lib/game/base_game.dart

```diff
@@ -42,18 +42,18 @@ class BaseGame extends Game with TapDetector {
       components.where((c) => c is Tapable).cast();
 
   @override
-  void onTapCancel() {
-    _tapableComponents.forEach((c) => c.handleTapCancel());
+  void onTapCancel(int pointerId) {
```

Comment:
wait I didn't notice this change before, this is total breaking change? @erickzanardo

Reply:
Its like I mentioned before on Telegram...

I have added multi touch support for the tapable components as well, this is a minor breaking change, because now BaseGame extends `MultiTouchTapDetector` instead of `TapDetector`.

I don't people will be affected by this change, since the `MultiTouchTapDetector` is a new mixin, nobody uses it, the only "weird" effect that this change will create is that people who uses the `TapDetector` mixin on their BaseGame class, will no more have the problem on #199 because now BaseGame uses other mixin.